### PR TITLE
[10.0][MIG] partner_financial_risk

### PR DESCRIPTION
**Pull request by Ehtaga**
Split of the #442 PR

### CODE REVIEW

**Comment by pedrobaeza** [comment]
@carlosdauden @chienandalu  can you check?

File: partner_financial_risk/README.rst [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+If any limit is exceeded, you won't be able to confirm any of its invoices
+unless you are authorized (Account Adviser group).
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+#. Go to *Invoicing/Accounting > Configuration > Settings > Invoicing & Payments*
```
Comment (chienandalu) [inline partner_financial_risk/README.rst:19]: *Invoicing/Accounting > Configuration > Settings > Accounting & Finance*

File: partner_financial_risk/README.rst [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+#. In the *Financial Risk* section, fill *Unpaid Margin* for setting the number
```
Comment (chienandalu) [inline partner_financial_risk/README.rst:20]: *Unpaid Margin* -> *Maturity Margin*

**Review by chienandalu — CHANGES_REQUESTED** [review]
- When I set a limit to Draft Invoices no warning raises if the limit is passed in a new invoice.
- When editing the customer Financial Risk, this is how the settings look like:
image

- Looks like this module test comments is causing travis fail (https://github.com/OCA/partner-contact/blob/10.0/partner_alias/tests/test_partner_alias.py#L20)

**Comment by chienandalu** [comment]
@Ehtaga you may be interested in this fix for v9: https://github.com/OCA/partner-contact/pull/468/files

**Comment by pedrobaeza** [comment]
Superseeded by #470

**Pull request closed without merging**